server/services/ImagineService.js

```diff
@@ -29,12 +29,18 @@ const submitStudy = async (data) => {
   }
 };
 
+const determineGroupSimple = async () => {
+  const options = ['experiential', 'expression', 'control'];
+  const randIndex = Math.floor(Math.random() * options.length);
+  return options[randIndex];
+}
+
```

Comment:
This looks good! I could see us extending this function to be able to pass in as a prop the group options that you want, for future imagine activities